src/app/components/modules/simplified-lunch/simplified-lunch.component.scss

```diff
@@ -0,0 +1,269 @@
+@import '../../../../styles/settings';
+
+/* --------------------------
+  Main Styles
+--------------------------- */
+.module-container{
+  width:100%;
+  height:100%;
+}
+.module-container[max-height~='500px']{
+  .not-main-entree{
+    display: none;
+  }
+  .lunch-category{
+    border-top: none;
+  }
+}
+.module-container[max-width~='500px']{
+  .not-main-entree{
+    display: none; 
+  }
+  .lunch-category{
+    border-top: none;
+  }
+}
+.module-container[max-height~='300px']{
+  .lunch-date{
+    display:none;
+  }
+  .lunch-category{
+    padding-top: 0rem;
+  }
+
+}
+
+.lunch-week .lunch-category{
+  border-top: 1px solid rgba(white, 0.5);
+
+}
+.loading-container {
```

Comment:
Generally, HTML styles are placed in the order in which they appear on the page. Consider moving `.loading-container` to the very top, right under `.module-container` and its respective element queries (because `.module-container` is the very first element). If there's any other styles that might need to be reorganized, do that too